content.js

```diff
@@ -78,7 +81,10 @@ function findCard() {
         thumbnail.src = result.thumbnailProperties.thumbnail
 
         const title = target.querySelector('#video-title')
-        const channelName = target.querySelector('.ytd-channel-name a')
+        let channelName = target.querySelector('.ytd-channel-name a')
+        if (!channelName) {
```

Comment:
Channel name doesn't include a link as child node on video page suggestions, hence this fallback